sage-ios/SAGE/SAGE/BrowseMentorsTableViewCell.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  BrowseMentorsTableViewCell.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 11/9/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+
+class BrowseMentorsTableViewCell: UITableViewCell {
+    
+    var mentorPicture = UIImageView()
+    var mentorName = UILabel()
+    var schoolName = UILabel()
+    var totalHours = UILabel()
+    let user: User
+    
+    init(user: User) {
+        self.user = user
+        super.init(style: .Default, reuseIdentifier: "BrowseCell")
+    }
+    
+    override func layoutSubviews() {
+        
+        let imageURL = NSURL(string: user.imgURL!)
```

Comment:
filling ui contents should not go in layoutSubviews. put this in configureWithUser(_:) instead